Description: Change default XSane umask from 0007 to 0077
 A default umask of 0007 can be mildly insecure in a multiuser environment,
 so tighten things up a bit and go with 0077 instead.
Author: Adrien Thebo
Bug-Debian: http://bugs.debian.org/592972
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/xsane/+bug/611950

Index: xsane-0.998/src/xsane.h
===================================================================
--- xsane-0.998.orig/src/xsane.h	2011-02-04 19:50:32.653016002 +0100
+++ xsane-0.998/src/xsane.h	2011-06-02 15:26:56.869664052 +0200
@@ -104,7 +104,7 @@
 #define XSANE_DEBUG_ENVIRONMENT	"XSANE_DEBUG"
 
 #define XSANE_PROGRESS_BAR_MIN_DELTA_PERCENT 0.025
-#define XSANE_DEFAULT_UMASK		0007
+#define XSANE_DEFAULT_UMASK		0077
 #define XSANE_HOLD_TIME			200
 #define XSANE_CONTINUOUS_HOLD_TIME	10
 #define XSANE_DEFAULT_DEVICE		"SANE_DEFAULT_DEVICE"
